packages/components/TestimonialSection.tsx

```diff
@@ -13,10 +10,8 @@ interface Props {
 }
 
 export default function TestimonialSection({ testimonials }: Props) {
-  const bgColor = useColorModeValue("bg-gray-50", "bg-gray-900");
-
   return (
-    <Section className={bgColor} innerClassName="grid gap-4">
+    <Section className="bg-gray-50 dark:bg-gray-900" innerClassName="grid gap-4">
```

Comment:
For a future improvement i think we should consider semantic tokens. [example (im not saying to implement shadcn!)](https://ui.shadcn.com/docs/theming#convention).
So it would be just `bg-background` and css would take care of rest.

Reply:
Good tip! I'll consider that for the backlog.

The next story we need to take is #152, but I think this is a nice improvement to have 👍🏼